lib/fork.js

```diff
@@ -134,14 +134,16 @@ module.exports = function (file, opts) {
 		isReady = true;
 	});
 
-	promise.run = function () {
+	promise.run = function (options) {
+		options = options || {};
```

Comment:
Yes. Tests aside there's only one place the "run chain" is called and it's from the Api.